clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -44,7 +42,11 @@ export class CopybookName {
 
 const experimentTag = "experiment-tag";
 export class CopybookDownloadService implements vscode.Disposable {
-  public static checkWorkspace(): boolean {
+  public storagePath: string;
+  public constructor(globalStoragePath: string) {
+    this.storagePath = globalStoragePath;
+  }
```

Comment:
```suggestion
  public constructor(public storagePath: string) { }
```

Reply:
should be resolved : 9be6c0a07652acd9800635c19569de87906bf6ce